### ISSUE TITLE
7zip archive support

### ISSUE THREAD

**Issue by breadcat** [just-install/just-install#241]
I've been trying to add a few applications (PRs incoming) and have noticed that there's no support for extracing 7zip archives in the zip kind.
This feature would be nice for extracting mpv (https://mpv.srsfckn.biz/) which is only distributed in .7z archives.

Is it possible to add support?

**Comment by geek1011** [comment]
Good idea. I am quite busy right now, but I will look into implementing this.

**Comment by lvillani** [comment]
I wonder if we could trigger installation of 7zip via just-install if missing and then use its command line tool to extract the archive somewhere. I cannot find a pure-Go library to extract a `.7z` archive.

**Comment by breadcat** [comment]
A potential workaround would be to extract the stand-alone 7za binary to the just-install temporary folder on first .7z application request.
Annoyingly, they only package the [latest command line version](http://www.7-zip.org/a/7z1604-extra.7z) as a .7z archive, so if going down this path we could use the [older 9.20 release](http://www.7-zip.org/a/7za920.zip) which is a plain zip file and would still do the job (as far as I know).
